src/loan_form/LoanEntries.jsx

```diff
@@ -1,10 +1,15 @@
 import React from 'react'
 import LoanEntry from './loan_entry/LoanEntry'
 
-export default function LoanEntries({ loans }) {
-  return (
-    <div id='loan-entries'>
-      { loans.map((loan, idx) => <LoanEntry key={idx} {...loan} />) }
-    </div>
-  )
+export default class LoanEntries extends React.Component {
```

Comment:
what is the reason that we've made this a class?

Reply:
For some reason when it was a function I wasn't able to pass handleChange through, so I eventually gave up and switched it to a class since I knew how to do it that way. I assume there's a way to keep it a function

Reply:
huh... i'll try to look into this before saturday

Reply:
nvm i figured it out, i was just being dumb